File: test/htdocs/test.example.org/009.php
<!DOCTYPE html>
<html>
<head>
<meta charset="ISO-8859-1">
<title>HTML/2.0 Test File: 009</title>
</head>
<body>
<?php
set_time_limit(0);
	echo "<h1>HTML/2.0 Test File: 009</h1>";
	echo "This page shows a timestamp every 1 second from the server.<br>";
	echo "--------------------------------------------------------------------------------<br>";
	flush();
	ob_flush();
	try{
		while (TRUE){
			echo date('m-d-Y H:i:s')."<br>";
			sleep ( 1 );
			flush();
			ob_flush();
		}
	} catch(Exception $e){
		// Nothing to do
	}
?>
</body>
</html>

File: test/htdocs/test.example.org/009.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import cgi, sys, time
import cgitb; cgitb.enable()

print "Content-Type: text/html;charset=UTF-8"
print

print """\
	<!DOCTYPE html><html><head>
	<title>HTML/2.0 Test File: 008 (server time)</title></head>
	<body><h1>HTML/2.0 Test File: 008</h1>"""

for i in range(60):
	s = time.strftime("%Y-%m-%d %H:%M:%S")
	print "<div>", s, "</div>"
	sys.stdout.flush()
	time.sleep(1)

print "</body></html>"